pkg/templates/csi.go

```diff
@@ -73,17 +74,42 @@ const CSIOperatorConfigName = "ceph-csi-operator-config"
 
 var CSIOperatorConfigSpec = csiopv1a1.OperatorConfigSpec{
 	DriverSpecDefaults: &csiopv1a1.DriverSpec{
+		Log: &csiopv1a1.LogSpec{
+			Rotation: &csiopv1a1.LogRotationSpec{
+				Periodicity: csiopv1a1.DailyPeriod,
+				MaxLogSize:  resource.MustParse("500M"),
+				MaxFiles:    7,
```

Comment:
do we define this as a constants values?

Reply:
I have no preference of one over other however I have my reservations on when to use const, mostly for invariants across many releases, env var, cross-pkg knowledge etc and I don't deem above should be a const.